testdata/pretty-microerror-1-depth-annotation-stack-trace.golden

```diff
@@ -0,0 +1,3 @@
+Error: Something went wrong: Something bad happened, and we had to crash
+
+--REPLACED--/pretty_test.go:161
```

Comment:
For me it doesn't make any difference but maybe that would look better:

```
Error: Something went wrong: something bad happened, and we had to crash
```

```
Error: Not found: cluster CR
```

vs


```
Error: Something went wrong: Something bad happened, and we had to crash
```

```
Error: Not found: Cluster CR
```

Reply:
I kinda like having the annotation lowercase, as well. I'll make the change